mlrun/runtimes/function.py

```diff
@@ -659,19 +659,23 @@ def with_node_selection(
             )
         super().with_node_selection(node_name, node_selector, affinity, tolerations)
 
-    @min_nuclio_versions("1.8.1")
+    @min_nuclio_versions("1.8.6")
     def with_preemption_mode(self, mode):
         """
         Preemption mode controls whether pods can be scheduled on preemptible nodes.
         Tolerations, node selector, and affinity are populated on preemptible nodes corresponding to the function spec.
 
-        Three modes are supported:
+        Four modes are supported:
```

Comment:
Switch to wording that requires less maintenance, something like
"The supported modes are:"